### PR TITLE
Improvement/back end for logging

### LINKED ISSUE #15: Create functionality of the Login panel

### PR DESCRIPTION
**Pull request by Cialkowski**
create new class with databse queries, create logging in functionality

### CODE REVIEW

File: UberBarber/UberBarber/View/LoginView.xaml.cs [hunk 1]
```diff
@@ -42,7 +44,11 @@ private void btnClose_Click(object sender, RoutedEventArgs e)
 
         private void btnLogin_Click(object sender, RoutedEventArgs e)
         {
-            //TODO
```
Comment (Borzygniew) [inline UberBarber/UberBarber/View/LoginView.xaml.cs:45]: Todo deleted, where docstring?

File: UberBarber/UberBarber/database/DatabaseQueries.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            /// <summary> This function opens connection to server and databse and executes logging query. </summary>>
+            /// <returns> True if authentication and query execute is successful, otherwise False. </returns>
+
+            try
+            {
+                database._connection.Open();
+            }
+            catch (MySqlException)
+            {
+
```
Comment (Borzygniew) [inline UberBarber/UberBarber/database/DatabaseQueries.cs]: remove unnecessary blank lines

**Review by Borzygniew — CHANGES_REQUESTED** [review]
except two little mistakes, whole pull request looks nice

File: UberBarber/UberBarber/View/LoginView.xaml.cs (hunk 1 above)
Comment (Cialkowski) [reply UberBarber/UberBarber/View/LoginView.xaml.cs:45]: done

File: UberBarber/UberBarber/database/DatabaseQueries.cs (hunk 2 above)
Comment (Cialkowski) [reply UberBarber/UberBarber/database/DatabaseQueries.cs]: done

**Review by Borzygniew — APPROVED** [review]
Zrzut ekranu 2023-01-11 213744

**Review by goflyy — APPROVED** [review]

**Pull request merged**